docs/src/RunningOnGPUs.md

```diff
@@ -0,0 +1,45 @@
+# Running on GPUs
+
+ITensor provides package extensions for running tensor operations in the GPU with different backends.
+You can activate a backend by loading the Julia GPU package alongside ITensors.jl
+and moving your tensors to the GPU using that package's provided accelerator.
+
+For example, you can load CUDA.jl 8 to perform tensor operations on NVIDIA GPUs or Metal.jl 3 to perform tensor operations on Apple GPUs:
```

Comment:
What are the "8" and "3" here referring to?

Reply:
Oh, these are the package versions. I agree that it is better to them out.